### PR TITLE
Add support for highflow usb

### PR DESCRIPTION
**Pull request by leoratte**
### Checklist:

- [x] My code follows Linux code style guidelines
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested my changes on the affected device(s) - tested on highflow usb

### CODE REVIEW

File: aquacomputer_d5next.c
Comment (aleksamagicka) [inline aquacomputer_d5next.c:1]: Highflow should also be noted in the header comment.

**Review by aleksamagicka — CHANGES_REQUESTED** [review]
Nice, thanks! A few comments enclosed.

**Comment by leoratte** [comment]
Addressed your comments and added reading firmware version and serial number which were missing.

File: re-docs/highflow/highflow_sensors.hexpat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+le u16 firmware @ 0x3;
+le u16 serial_number @ 0x9;
+
+le u16 raw_pressure @ 0x11;
```
Comment (aleksamagicka) [inline re-docs/highflow/highflow_sensors.hexpat:4]: Are these pressure values useful? Any reason for not exposing them? If they are useful, but require something else from Aquasuite that we can't do from the driver, that should be noted in the docs.

Comment (leoratte) [reply re-docs/highflow/highflow_sensors.hexpat:4]: I don't think they are useful for the high flow usb sensor. They are probably related to the mps pressure sensors.

Comment (aleksamagicka) [reply re-docs/highflow/highflow_sensors.hexpat:4]: Then that's something we'll have to verify in #89, if we get some info from there. Thanks!

**Review by aleksamagicka — APPROVED** [review]

**Comment by aleksamagicka** [comment]
I've just found that LibreHardwareMonitor has MPS support implemented (#89), and all offsets look [exactly the same](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/master/LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/MPS.cs), including the [product ID](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/master/LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/AquaComputerGroup.cs#L47). Seems like we have both covered now.

**Pull request merged**